server/@types/express/index.d.ts

```diff
@@ -7,6 +8,7 @@ declare module 'express-session' {
   interface SessionData {
     returnTo: string
     nowInMinutes: number
+    availableSupportTypes: SupportType[]
```

Comment:
On first request, the list of available `SupportType` options (names and descriptions) is retrieved from the visit scheduler and stored here.